azure-pipelines.yml

```diff
@@ -49,6 +53,15 @@ steps:
     DOCKER_OVERRIDE: $(dockerOverride)
     dockerHubUsername: $(dockerHubUsername)
     dockerHubImageName: $(imageName)
+- script: |
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c "bundle exec rake cc:report"
```

Comment:
do you need to add it here as well?

Reply:
This is a report task `cc:report`, earlier one was a setup task `cc:setup`. Reporter sends a success or fail message to Code Climate